_posts/0000-01-02-yousifkeasou.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Our initial research indicates that the exact percentage/number of legal firms that use chatbots is hard to come by. However, there's plenty of information on the use of chatbots in the legal industry as well as dozens of examples of successful legal chatbots. Hence, we'd recommend continuing with this research by providing 5-7 brief uses/benefits of using chatbots in the legal industry. These benefits may expand on the benefits listed in the initial finding